src/js/common/schemaform/save-load-actions.js

```diff
@@ -282,3 +309,52 @@ export function fetchInProgressForm(formId, migrations, prefill = false) {
     });
   };
 }
+
+export function removeInProgressForm(formId, migrations) {
+  return (dispatch, getState) => {
+    const userToken = sessionStorage.userToken;
+    const trackingPrefix = getState().form.trackingPrefix;
+    // If we don't have a userToken, fail safely
+    if (!userToken) {
+      dispatch(setFetchFormStatus(LOAD_STATUSES.noAuth));
```

Comment:
Yeah, you shouldn't get here, it's carried over from the other actions. Might be able to take it out here, since it seems pretty unlikely (and would get caught with a 401 response later).